lib/widgets/menu/custom_alert.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+import '../../shared/ui_consts.dart';
+
+class CustomAlert extends StatelessWidget {
```

Comment:
I'd rename this as `ExitAlert`, as it's its only task. `CustomAlert` seems too generic for this specific widget

Reply:
Done